ui/dashboards/src/components/Variables/VariableEditorForm/VariableEditorForm.tsx

```diff
@@ -159,6 +209,11 @@ export function VariableEditForm({
           <SectionHeader>List Options</SectionHeader>
           <Grid container spacing={2} mb={2}>
             <Grid item xs={6}>
+              {/** Hack?: Cool technique to refresh the preview to simulate onBlur event */}
```

Comment:
This can be used to reload the preview whenever something is clicked outside of the Inputs
More details here: https://www.loom.com/share/9b978d45503042c1a55dd34d7b182762